subsys/net/openthread/rpc/server/ot_rpc_cli.c

```diff
@@ -68,10 +68,14 @@ NRF_RPC_CBOR_CMD_DECODER(ot_group, ot_rpc_cmd_cli_init, OT_RPC_CMD_CLI_INIT, ot_
 static void ot_rpc_cmd_cli_input_line(const struct nrf_rpc_group *group,
 				      struct nrf_rpc_cbor_ctx *ctx, void *handler_data)
 {
-	char input_line_buffer[256];
 	struct nrf_rpc_cbor_ctx rsp_ctx;
 	char *result;
 
+#if CONFIG_OPENTHREAD_CLI_MAX_LINE_LENGTH > 256
```

Comment:
Shouldn't it be allocated from heap instead? 
This fix seems like we want to avoid memory misconfiguration.
@Damian-Nordic, opinions?